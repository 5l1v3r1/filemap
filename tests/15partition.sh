#!/bin/sh

fm -vvv map -qi '/test/*' "split.sh <"
fm -v ls -1 '/test/*/*split.sh'
