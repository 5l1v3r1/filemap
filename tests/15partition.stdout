/test/f1.d/split.sh/.status
/test/f1.d/split.sh/.stderr
/test/f1.d/split.sh/1.part
/test/f1.d/split.sh/4.part
/test/f1.d/split.sh/7.part
/test/f2.d/split.sh/.status
/test/f2.d/split.sh/.stderr
/test/f2.d/split.sh/1.part
/test/f2.d/split.sh/4.part
/test/f2.d/split.sh/7.part
